src/EPR.RegulatorService.Frontend.UnitTests/Web/Controllers/ReprocessorExporter/Registrations/RegistrationControllerTests.cs

```diff
@@ -0,0 +1,75 @@
+using System.Threading.Tasks;
+
+using EPR.RegulatorService.Frontend.Core.Enums;
+using EPR.RegulatorService.Frontend.Core.Sessions;
+using EPR.RegulatorService.Frontend.Web.Controllers.ReprocessorExporter.Registrations;
+using EPR.RegulatorService.Frontend.Web.Sessions;
+using EPR.RegulatorService.Frontend.Web.ViewModels.ReprocessorExporter;
+
+using FluentAssertions;
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Moq;
```

Comment:
Remove unused using statements:

```suggestion
using EPR.RegulatorService.Frontend.Core.Enums;
using EPR.RegulatorService.Frontend.Core.Sessions;
using EPR.RegulatorService.Frontend.Web.Controllers.ReprocessorExporter.Registrations;
using EPR.RegulatorService.Frontend.Web.Sessions;
using EPR.RegulatorService.Frontend.Web.ViewModels.ReprocessorExporter;

using Microsoft.AspNetCore.Mvc;
using Microsoft.Extensions.Configuration;
```

Reply:
done